bin/run-ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
+set -o nounset
+SCALA_VERSION="$1"
+
+sbt -Dfile.encoding=UTF-8 \
+  -J-XX:ReservedCodeCacheSize=256M \
+  -J-Xmx3046M -J-Xms3046M -J-server \
+  zincRoot/test:compile scalafmtCheck \
+  "publishBridgesAndSet $SCALA_VERSION" \
+  crossTestBridges zincRoot/test zincRoot/scripted
```

Comment:
Could you put each distinct command on a separate line? It makes it easier to read and less scary to configure the CI.